fe/src/main/java/org/apache/doris/planner/SingleNodePlanner.java

```diff
@@ -802,7 +802,6 @@ private PlanNode createConstantSelectPlan(SelectStmt selectStmt, Analyzer analyz
         for (int i = 0; i < resultExprs.size(); ++i) {
             SlotRef slotRef = new SlotRef(tupleDesc.getSlots().get(i));
             resultExprs.set(i, slotRef);
-            selectStmt.getBaseTblResultExprs().set(i, slotRef);
```

Comment:
Is it OK when executing `insert into` statement?
I remember it seems to be introduced here in support of `insert into values()`. You can try if this patch works properly.

Reply:
Has reverted this change.